src/js/auth.js

```diff
@@ -34,11 +53,16 @@ document.addEventListener("DOMContentLoaded", function () {
                 }
                 return response.json();
             })
-            .then((data) => {
-                console.log("Registration successful:", data);
+            .then(() => {
+                username.value = "";
+                email.value = "";
+                password.value = "";
+                avatar.value = "";
+                banner.value = "";
+                alert("Registration successful!");
             })
             .catch((error) => {
-                console.error("Registration error:", error);
+                alert("Registration failed. Please check your inputs.");
```

Comment:
It is better UX to display the error (or success) message in the HTML rather than using an alert.